parser/src/parser.cpp

```diff
@@ -78,18 +142,10 @@ int main(int argc, char* argv[])
   po::store(po::command_line_parser(argc, argv)
     .options(desc).allow_unregistered().run(), vm);
 
-  std::string binDir = boost::filesystem::canonical(
-    boost::filesystem::path(argv[0]).parent_path()).string();
-  std::string pluginDir = binDir + "/../lib/parserplugin";
-
-  //--- Write out labels into a file to the data directory. ---//
-
-  if (vm.count("label"))
-  {
-    std::ofstream fLabels(vm["data-dir"].as<std::string>() + "/labels.txt");
-    for (const std::string& label : vm["label"].as<std::vector<std::string>>())
-      fLabels << label << std::endl;
-  }
+  if (!vm.count("input"))
+    vm.insert(std::make_pair(
```

Comment:
No it will not unless default value is provided in the arg. parser. Unfortunately a vector can't have a default value because the boost can't generate a help message from it.